rudder-agent/SPECS/rudder-agent.spec

```diff
@@ -387,7 +387,7 @@ then
   CFRUDDER_FIRST_INSTALL="true"
 fi
 
-/opt/rudder/share/package-scripts/rudder-agent-postinst "${CFRUDDER_FIRST_INSTALL}" "%{?_os}" "%{use_systemd}" ""
+/opt/rudder/share/package-scripts/rudder-agent-postinst "${CFRUDDER_FIRST_INSTALL}" "rpm" "%{use_systemd}" ""
```

Comment:
Why rpm here ?

Reply:
even if the option is currently ignored, rpm is confusing

Reply:
Ok, actually we are already using this syntax on the debian build so it is coherent.